### PR TITLE
Configure Cargo.toml

### PR DESCRIPTION
**Pull request by alteous**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -2,12 +2,23 @@
...
 authors = ["Benjamin Wasty <benny.wasty@gmail.com>"]
+description = "Mikkelsen tangent space algorithm"
+documentation = "https://docs.rs/mikktspace"
+repository = "https://github.com/bwasty/mikktspace-rs"
+homepage = "https://github.com/bwasty/mikktspace-rs"
+readme = "README.md"
+keywords = ["3D", "graphics", "algorithm", "tangent"]
+license = "MIT/Apache-2.0"
 build = "build.rs"
+exclude = ["examples/cube.obj", "examples/generate.c"]
```
Comment (bwasty) [inline Cargo.toml]: Perhaps even `examples/**/*`?

File: Cargo.toml [hunk 2]
```diff
@@ -2,12 +2,23 @@
 name = "mikktspace"
 version = "0.1.0"
 authors = ["Benjamin Wasty <benny.wasty@gmail.com>"]
```
Comment (bwasty) [inline Cargo.toml]: You could add yourself too...

**Review by bwasty — APPROVED** [review]

File: Cargo.toml (hunk 1 above)
Comment (alteous) [reply Cargo.toml]: Wouldn't that exclude generate.rs?

Comment (bwasty) [reply Cargo.toml]: Yes - I'd only include files necessary for library usage. I don't think many people would go looking for examples in the cryptic path somewhere under `~/.cargo`.
Unrelated: you could now replace `github.com/bwasty` with `github.com/gltf-rs` everywhere.

Comment (alteous) [reply Cargo.toml]: Oh, I understand now. I thought that excluding `generate.rs` would break the example but that's not the case.

**Comment by alteous** [comment]
I've changed the README file a bit.

**Comment by alteous** [comment]
Merging to see how it looks on the homepage.

**Pull request merged**